### PR TITLE
Development Guide

### PR DESCRIPTION
**Pull request by warpdragon**
Hoping to create a kickass development guide.

### CODE REVIEW

**Comment by imaGecko** [comment]
should check in with iburn who already started working on some documentation, forgot how much you already talked with him

**Comment by warpdragon** [comment]
send him on

**Comment by warpdragon** [comment]
I'm satisfied. This will be good. We can have our field-expert people put their notes in their departments' folders and insert links into the README link table.

File: docs/README.md [hunk 1]
```diff
@@ -1,26 +1,80 @@
-# Intro to Documentation
+<head>
 
-Documentation involves the creation, organization, and communication of information for the use of developers, players, and the general audience. This documentation takes various forms, from developmental resources to official documentation
+</head>
```
Comment (Chronophylos) [inline docs/README.md]: what are the `head` tags suppose to do?

File: docs/README.md [hunk 2]
```diff
@@ -1,26 +1,80 @@
...
+# OAA Development Guide
 
-It's fine to refer new contributors to http://warpdragon.github.io (The Documentation Hub) for resources. This site is just something I created because myself and others were finding our Bookmarks folder getting out-of-hand with all the links we needed to do our jobs. This website has, among other things, references for learning LUA, lists of useful Githubs, lots of the Artwork for the game, and tons of useful charts and graphs. Anyone can recommend adding things to this site.
+Updated 2017-12-05
 
-## The /docs/ folder
+## Introduction
 
-These documents should only be "official" information directly related to the development, release, and support of the OAA project. Think of it this folder as a Library, ensuring that everyone has easy access to the most current information available. To that end, those who contribute are also asked to seek out and make contacts on the various teams, and work with key members to ensure that notes are taken and reported whenever ideas or designs become "official".
+Describe how the document is organized. Explain what the reader should expect to get out of reading it:
```
Comment (Chronophylos) [inline docs/README.md]: maybe actually write something here.

File: docs/README.md [hunk 3]
```diff
@@ -1,26 +1,80 @@
...
 
-## Ways to contribute
+2. [Git][3]
 
-1. Open an issue at http://github.com/OpenAngelArena/oaa/issues. Tag your issue [Links], [Tables], [Instructionals], [Resource], etc please.
-2.  Create your own development branch (preferred method) and send a pull request to OpenAngelArena/oaa
+3. [Dota 2 Tools][4]
 
-## Questions?
+4. [D2AM][5]
```
Comment (Chronophylos) [inline docs/README.md]: `d2am` not `D2AM`

File: docs/README.md (hunk 1 above)
Comment (warpdragon) [reply docs/README.md]: css?

Comment (Chronophylos) [reply docs/README.md]: why? it's a markdown file it doesn't need something like this

Comment (warpdragon) [reply docs/README.md]: k

**Comment by warpdragon** [comment]
Really Travis? What did I ever do to you!?

**Comment by warpdragon** [comment]
Ready to be merged as soon as you fix Travis's broken shit.

**Pull request merged**